### DISCUSSION TITLE
libslepc.so.3.019 not found

### DISCUSSION THREAD

**Discussion by hhy2022** [idaholab/moose#28123] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] I have consulted the [Posting Guidelines](https://github.com/idaholab/moose/discussions/18270).
- [X] I have searched the [Discussions Forum](https://github.com/idaholab/moose/discussions) and [MOOSE Framework Troubleshooting](https://mooseframework.inl.gov/help/troubleshooting.html) and have not found what I was looking for
- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)

### Issue or question about MOOSE

Hi, I am trying to update MOOSE and everything goes well. However, when I am trying to compiling it, it gave me a message saying that "libslepc.so.3.019" not found. 
My test also didn't work.
Do you have any idea on how to fix it?
Thank you very much for any help.

**Comment by GiudGiud** [comment, marked as answer]
Hello

You ll need to git clean -xfd the repository

Some left over compiled object is looking for the previous version of petsc you had installed 

Guillaume

**Reply by hhy2022** [reply]
Thank you very much for the help. I have deleted the former version and re-install it. Everthing works well now.
